app/components/gameForm.tsx

```diff
@@ -17,26 +18,28 @@ export const GameForm = ({ onSubmit }: GameFormProps) => {
         console.log("e", e);
       }}
     >
-      <label>
-        Name:
-        <input
-          value={value.name}
-          type="text"
-          name="name"
-          onChange={e => onChange({ ...value, name: e.target.value })}
-        />
-      </label>
-      <label>
-        Surname:
-        <input
-          value={value.surname}
-          type="text"
-          name="surname"
-          onChange={e => onChange({ ...value, surname: e.target.value })}
-        />
-      </label>
+      <Label primary={false}>
```

Comment:
```suggestion
      <Label primary={false}>
```
Styled-Component daje nam możliwość dodawania własnych nazw dla stylowanych elementów. Dzięki temu możemy nazywać je tak, żeby mówiły co stylujemy semantycznie, np: <label> jest dookoła informacji o nowym graczu, więc można zamiast "Label" nadać nazwę "NewPlayerLabel"